src/public/style/explore-budget.scss

```diff
@@ -167,96 +166,38 @@ $F9-color: $area-color-10;
 
                 button{
                     padding: 0.5em;
-                    border: 1px solid #AAA;
+                    border: 1px solid $table-border-color;
 
                     &.selected{
-                        background-color: $area-color-1;
+                        background-color: $brand-color;
                     }
                 }
             }
         }
 
-        .M52-DF-0 path,
-        .M52-DI-0 path{
-            fill: $F0-color;
-        }
-        .M52-DF-1 path,
-        .M52-DI-1 path{
-            fill: $F1-color;
-        }
-        .M52-DF-2 path,
-        .M52-DI-2 path{
-            fill: $F2-color;
-        }
-        .M52-DF-3 path,
-        .M52-DI-3 path{
-            fill: $F3-color;
-        }
-        .M52-DF-4 path,
-        .M52-DI-4 path{
-            fill: $F4-color;
-        }
-        .M52-DF-5 path,
-        .M52-DI-5 path{
-            fill: $F5-color;
-        }
-        .M52-DF-6 path,
-        .M52-DI-6 path{
-            fill: $F6-color;
-        }
-        .M52-DF-7 path,
-        .M52-DI-7 path{
-            fill: $F7-color;
-        }
-        .M52-DF-8 path,
-        .M52-DI-8 path{
-            fill: $F8-color;
-        }
-        .M52-DF-9 path,
-        .M52-DI-9 path{
-            fill: $F9-color;
-        }
-
-        .legend-list .color{
-            &.M52-DF-0,
-            &.M52-DI-0{
-                background-color: $F0-color;
-            }
-            &.M52-DF-1,
-            &.M52-DI-1{
-                background-color: $F1-color;
-            }
-            &.M52-DF-2,
-            &.M52-DI-2{
-                background-color: $F2-color;
-            }
-            &.M52-DF-3,
-            &.M52-DI-3{
-                background-color: $F3-color;
-            }
-            &.M52-DF-4,
-            &.M52-DI-4{
-                background-color: $F4-color;
-            }
-            &.M52-DF-5,
-            &.M52-DI-5{
-                background-color: $F5-color;
-            }
-            &.M52-DF-6,
-            &.M52-DI-6{
-                background-color: $F6-color;
+        @for $i from 0 to length($area-colors) {
```

Comment:
J’aurais utilisé `@for $i from 1 though length($area-colors)` pour éviter de faire des `$i+1` partout dans le bloc de la boucle.

Reply:
Y'a les deux (`$i` et `$i+1`) :-(

J'ai gardé cette forme par consistence avec le premier bloc et avec les `.map()`/`.forEach()` JavaScript.

Reply:
ok!